src/js/scrollArea.jsx

```diff
@@ -148,6 +163,11 @@ export default class ScrollArea extends React.Component{
         this.setState(newState);
     }
 
+    convertLinesToPixels(lines) {
+        var pixels = lineHeight(findDOMNode(this.refs.content));
```

Comment:
It is necessary, because each handleWheel event tells the exact number of lines that were scrolled, which depends how fast the wheel was moved. The value may be a number like 3.4365.